lib/rubocop/cli.rb

```diff
@@ -233,7 +232,6 @@ def print_cop_details(cops)
         puts '# Supports --auto-correct' if cop.new.support_autocorrect?
         puts "#{cop.cop_name}:"
         puts config_lines(cop)
-        puts
```

Comment:
This `puts` adds an empty line between under the other content. It might be helpful to leave it.